pandas/tests/indexes/test_base.py

```diff
@@ -2020,3 +2022,64 @@ def test_repeat(self):
         with tm.assert_produces_warning(FutureWarning):
             result = idx.repeat(n=repeats)
             tm.assert_index_equal(result, expected)
+
+    def test_is_monotonic_na(self):
+        examples = [pd.Index([np.nan]),
```

Comment:
`test_numeric.py` under the `Int64Index` tests (not sure why they were there in the first place).  This is just a refactoring i.e., I moved tests around (did not change them) when I was writing out the `UInt64Index` tests.